ecs-init/config/common.go

```diff
@@ -45,6 +46,23 @@ const (
 	// DefaultRegionName is the default region to fall back if the user's region is not a region containing
 	// the agent bucket
 	DefaultRegionName = endpoints.UsEast1RegionID
+
+	// dockerJSONLogMaxSize is the maximum allowed size of the
+	// individual backing json log files for the managed container.
+	dockerJSONLogMaxSize = "16m"
```

Comment:
whats the current size?

Reply:
There isn't a limit on the file size by default:

The option's default value is `-1`:
https://github.com/docker/docker-ce/blob/a70ecfc169d3149fba0392c8b26cb5ed1aecc6e3/components/engine/daemon/logger/jsonfilelog/jsonfilelog.go#L45-L55

Which means that the file won't be limited by the underlying utility:
https://github.com/docker/docker-ce/blob/a70ecfc169d3149fba0392c8b26cb5ed1aecc6e3/components/engine/daemon/logger/loggerutils/logfile.go#L169-L172